code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -30,11 +30,20 @@
 
 /obj/item/clothing/mask/facehugger/Initialize(mapload)
 	. = ..()
+	compile_monkey_icon()
```

Comment:
Isn't this already called in `/obj/item/Initialize()`?
The only condition being `ismonkey(user)`, which, I suppose if there's not a monkey with one roundstart it won't run, but doesn't it also do this when a monkey puts the mask on?

Reply:
For whatever reason, no. Not calling it doesn't work.
There's a good reason but I've since forgot.
![image](https://user-images.githubusercontent.com/40559528/193516583-b99e8177-293c-4e83-b590-94711c6cf119.png)

Reply:
Oh, the equip was overridden lol
![image](https://user-images.githubusercontent.com/40559528/193743322-65872d4e-0261-44c5-bfdf-8cfaeca09ed2.png)